### DISCUSSION TITLE
Donations / Patreon

### DISCUSSION THREAD

**Discussion by jellersby** [Acly/krita-ai-diffusion#482] [Ideas]
I'd like to start a discussion about compensating the developer(s) of this wonderful project for their time and effort. This is an important project that delighted users (such as myself) will hopefully be relying on for many years to come.
I think the project should support its developers, and the most likely source of funds is happy users. Is there a current known way to support this project? I've heard that OpenCollective may not be a viable platform anymore, but there may be other options, like Patreon.

Thank you Acly and contributors!

**Comment by zengels** [comment]
Agree . i would love to contribute !!

**Comment by lovemakerz** [comment]
Hi, 

It's Github, you can allready support Acly,

I agree with you, this project is exceptional, both in terms of quality and follow-up.

![image](https://github.com/Acly/krita-ai-diffusion/assets/19786231/e8af2041-627b-4e7a-876b-3c2265b5273d)
